postpic/datahandling.py

```diff
@@ -120,7 +120,12 @@ class -- the final output of the postpic postprocessor.
     unwrap_phase = None
 
 
-__all__ = ['Field', 'Axis']
+__all__ = ['KeepDim', 'Field', 'Axis']
+
+
+class KeepDim(object):
+    def __init__(self, value):
```

Comment:
Should that check, that value is indeed just a single float or int? Particularly `field[KeepDim(slice(0, 5))]` would need to either work or be excluded.

Reply:
Using a `slice` implies to keep the dimension, so KeepDim is redundant. I changed `Axis._normalize_slice` to make this case just work.